FileShareService.DesktopClient.Core/Jobs/NewBatchJob.cs

```diff
@@ -170,8 +171,16 @@ public class NewBatchJobParams
 
     public class NewBatchFiles
     {
+        private string expectedFileCount = string.Empty;
+
         public string SearchPath { get; set; } = string.Empty;
-        public int ExpectedFileCount { get; set; }
+
+        //ExpectedFileCount can be "*" or an integer number
+        public string ExpectedFileCount { 
+            get => expectedFileCount; 
+            set => expectedFileCount = value?.Trim() ?? string.Empty; 
```

Comment:
I don't think `value` can be null as it's not a `string?`

Reply:
Ok, thanks, I'll have to check how the JSON deserializer populates this field

Reply:
Having discussed with Chris I'm happy with this as is.